Dockerfile

```diff
@@ -48,9 +98,17 @@ COPY . ./
 #            CMD [ "/usr/bin/wget", "--no-verbose", "--tries=1", "--spider", "http://127.0.0.1:8080/openapi" ]
 
 # collect information from git repositories
-RUN /opt/recipes/venv/bin/python version.py
-# delete git repositories to reduce image size
-RUN find . -type d -name ".git" | xargs rm -rf
+RUN <<EOF
+  /opt/recipes/venv/bin/python version.py
+  # delete git repositories to reduce image size
+  find . -type d -name ".git" | xargs rm -rf
```

Comment:
I would add .git in the `.dockerignore` file instead of removing it. This is kinda meh since this only removes it from the final image the `.git` is already in the COPY layer which would be avoided when using the dockerignore.

Reply:
I agree. Removing files added in an earlier layer doesn't actually remove them from the image. The files are still there even if they aren't visible, so it doesn't reduce the container image's size.